pytorch_pfn_extras/onnx/pfto_exporter/export.py

```diff
@@ -735,7 +776,10 @@ def assign_onnx_values(
                     if n.kindOf(attr_name) == "t":
                         attr = onnx.helper.make_attribute(attr_name, _tensor_to_proto(n.t(attr_name)))
                     else:
-                        attr = onnx.helper.make_attribute(attr_name, n[attr_name])
+                        if pytorch_pfn_extras.requires('1.13'):
+                            attr = onnx.helper.make_attribute(attr_name, sym_hel._node_get(n, attr_name))  # type: ignore[attr-defined]
```

Comment:
Using `torch.onnx.symbolic_helper._node_get` instead since patching functions for `torch._C.Graph` would be removed